### PR TITLE
fix: build with Xcode 12.4 SQPIT-781

### PR DESCRIPTION
**Pull request by billypchan**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/SQPIT-781" title="SQPIT-781" target="_blank">TaskSQPIT-781</a>  Build frameworks compiles on both Xcode 12 and 13
  </td></table>
  <br />
 

# What's new in this PR?

ref: https://github.com/wireapp/wire-ios-protos/pull/60

### CODE REVIEW

**Comment by KaterinaWire** [comment]
Why were the images changed?

**Comment by billypchan** [comment]
@KaterinaWire the snapshot images are restored to iOS 14 version.

**Review by KaterinaWire — APPROVED** [review]

File: azure-pipelines.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ resources:
   - repository: wire-ios-shared-resources
     type: github
     name: wireapp/wire-ios-shared-resources
-    ref: refs/heads/master # Branch to fetch the jobs template from
+    ref: refs/heads/billypchan-patch-2 # Branch to fetch the jobs template from
```
Comment (billypchan) [inline azure-pipelines.yml:6]:
```suggestion
    ref: refs/heads/master # Branch to fetch the jobs template from
```

**Review by johnxnguyen — APPROVED** [review]

**Comment by billypchan** [comment]
release patch

**Comment by billypchan** [comment]
release patch

**Comment by billypchan** [comment]
release patch

**Comment by zenkins** [comment]
Hello, @billypchan! Merging PR for command **release patch** for PR #**48**. Will create a **patch** release after merged.
Next version is 14.0.3.

**Pull request merged**